src/sonic-yang-mgmt/tests/libyang-python-tests/config_data.json

```diff
@@ -97,7 +97,12 @@
 					"vlanid": 111,
 					"port": "Ethernet6",
 					"tagging_mode": "tagged"
-				}
+				},
+                                {
+                                        "vlanid": 111,
+                                        "port": "Ethernet8",
+                                        "tagging_mode": "tagged"
+                                }
```

Comment:
Could you fix the indentation of the whole file? We normally use 4 spaces instead of one tab.

Reply:
Still applicable